web/docs/guides/database/managing-passwords.mdx

```diff
@@ -8,11 +8,8 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 Your PostgreSQL database is the core of your Supabase project, so it's important that it has a strong, secure password at all times.
-We recommend avoiding these symbols: 
 
-`: / ? # ] [ @ ! $ & ' ( ) * + , ; =`
-
-Special symbols can conflict with connection string URLs, so it's best to avoid them. You can make a secure password using a longer string or a randomized set of upper and lowercase letters and digits.
+If you use special symbols in your postgres password, you must remember to percent encode your password later if using the postgres connection string e.g. `postgresql://postgres:p%3Dword@db.cvwawazfelidkloqmbma.supabase.co:5432/postgres`
```

Comment:
perhaps add a link for `[percent-encode](https://en.wikipedia.org/wiki/Percent-encoding)`?